frontend/src/features/search/SearchResultItem.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import numeral from 'numeral'
+import { Link } from 'react-router-dom'
+
+import DatasetInfoItem from '../dataset/DatasetInfoItem'
+import RelativeTimestamp from '../../chrome/RelativeTimestamp'
+
+import { SearchResult } from '../../qri/searchResult'
+
+interface SearchResultItemProps {
+  dataset: SearchResult
+}
+
+const SearchResultItem: React.FC<SearchResultItemProps> = ({ dataset}) => {
+
+    const {
```

Comment:
```suggestion
const SearchResultItem: React.FC<SearchResultItemProps> = ({ dataset }) => {
    const {
```

(add space on prop destructuring, drop empty line after func declaration)